From 9444607ab3dcf5fb8091bf894a65c1f161de580d Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 17 Jul 2026 15:35:17 +0200
Subject: [PATCH] tests: monitor: enclose device names in quotes

JIRA: https://issues.redhat.com/browse/RHEL-190549
Upstream Status: nftables commit 26746952952bba8c19aebbd03a55decbc0d0c5fc

commit 26746952952bba8c19aebbd03a55decbc0d0c5fc
Author: Pablo Neira Ayuso <pablo@netfilter.org>
Date:   Thu Jul 10 02:53:50 2025 +0200

    tests: monitor: enclose device names in quotes

    Update test to enclose flowtable device names in quotes, otherwise,
    it reports a spurious issue:

    @@ -1,2 +1,3 @@
     add table ip t
    -add flowtable ip t ft { hook ingress priority 0; devices = { lo }; }
    +add flowtable ip t ft { hook ingress priority 0; devices = { "lo" }; }

    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 tests/monitor/testcases/flowtable-simple.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/monitor/testcases/flowtable-simple.t b/tests/monitor/testcases/flowtable-simple.t
index b373cca..11254c5 100644
--- a/tests/monitor/testcases/flowtable-simple.t
+++ b/tests/monitor/testcases/flowtable-simple.t
@@ -1,6 +1,6 @@
 # setup first
 I add table ip t
-I add flowtable ip t ft { hook ingress priority 0; devices = { lo }; }
+I add flowtable ip t ft { hook ingress priority 0; devices = { "lo" }; }
 O -
 J {"add": {"table": {"family": "ip", "name": "t", "handle": 0}}}
 J {"add": {"flowtable": {"family": "ip", "name": "ft", "table": "t", "handle": 0, "hook": "ingress", "prio": 0, "dev": "lo"}}}
